### PR TITLE
WIP Refactor and clean code

### PR DESCRIPTION
**Pull request by spencerhandley**
This cleans up a lot of the preexisting code and splits a lot of the core functionality into classes in order to avoid crowding the global scope. It also implies a sensible architecture for the project that we can continue to build upon.

### CODE REVIEW

File: includes/class-patreon-metabox.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Author URI: http://patreon.com
+*/
+namespace patreon;
+
+if( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+class Patreon_Metabox {
+
+    public function patreon_plugin_meta_boxes_setup() {
+        debug_to_console("HEYYYY");
```
Comment (zsennenga) [inline includes/class-patreon-metabox.php]: I assume these'll get removed

**Comment by zsennenga** [comment]
Not familiar with the wordpress-ism, but def is WAY better structured PHP

File: includes/class-patreon-metabox.php (hunk 1 above)
Comment (spencerhandley) [inline includes/class-patreon-metabox.php]: yea, it's very in progress

File: admin/class-admin.php [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+Plugin Name: Patreon
+Plugin URI:
+Description: Stay close with the Artists & Creators you're supporting
+Version: 1.0
+Author: Ben Parry
```
Comment (derekr) [inline admin/class-admin.php]: 🤔

Comment (spencerhandley) [reply admin/class-admin.php]: Yep, that's the original author of our wordpress plugin.

**Pull request closed without merging**